imagick/imagick.go

```diff
@@ -153,10 +154,11 @@ func (ng Engine) GetImageInfo(b []byte, srcFormat ...string) (*imgry.ImageInfo,
 type Image struct {
 	mw *imagick.MagickWand
 
-	data   []byte
-	width  int
-	height int
-	format string
+	data          []byte
+	width         int
+	height        int
+	format        string
+	convertFormat string
```

Comment:
what is convertFormat used for...? perhaps call this, the destFormat ...? or outFormat .. ? endFormat ..?